src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleSearchChange = (event: { target: { value: string; }; }) => {
```

Comment:
Use react types for events in handlers

```suggestion
  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```